konig-core/src/main/java/io/konig/core/vocab/XOWL.java

```diff
@@ -0,0 +1,32 @@
+package io.konig.core.vocab;
+
+/*
+ * #%L
+ * Konig Core
+ * %%
+ * Copyright (C) 2015 - 2018 Gregory McFall
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+
+import org.openrdf.model.URI;
+import org.openrdf.model.impl.URIImpl;
+
+public class XOWL {
+	public static final URI TERMSTATUS = new URIImpl("http://schema.pearson.com/ns/xowl/termstatus");
+	public static final URI Stable = new URIImpl("http://schema.pearson.com/ns/xowl/termstatus/Stable");
+	public static final URI Experimental = new URIImpl("http://schema.pearson.com/ns/xowl/termstatus/Experimental");
```

Comment:
Please replace the terms `Stable` and `Experimental` with:

* statusStable
* statusExperimental

And add:
* statusSubmitted
* statusInvalid
* statusSuperseded
* statusRetired

These terms are defined in the document [Term Status Annotations](https://docs.google.com/document/d/17HRBbURSjLThnotRsez2YY0goCF9REb4LEELrpRs7k8/edit?usp=sharing).